bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/SyncThread.java

```diff
@@ -119,6 +119,12 @@ public void startCheckpoint(Checkpoint checkpoint) {
         });
     }
 
+    void start() {
```

Comment:
Can you rename this to "scheduleCheckpointAtFixedRate(long flushIntervalMs)" as I suggested in #1201?

- it is not actually a "start" action because the scheduler has been started, this is more a "schedule" action
- flushInterval doesn't belong to SyncThread any more.

Also can you update the comment about 'flushInterval' in ServerConfiguration and conf/bk_server.conf, `site/_data/config/bk_server.yaml`? `flushInterval` is now only works when `entryLogPerLedger` enabled.